code/game/turfs/simulated/minerals.dm

```diff
@@ -0,0 +1,500 @@
+/**********************Mineral deposits**************************/
```

Comment:
Are any spriters willing to isolate the clown and mime minerals in this file:

https://github.com/ParadiseSS13/Paradise/blob/master/icons/turf/smooth_wall.dmi

Basically, they need to have the asteroid background removed. Right now the mime one is really ugly on the mining scanner.